messages/nl/app.php

```diff
@@ -0,0 +1,200 @@
+    '{0, plural, =1{# minute} other{# minutes}} ago' => '{0, plural, =1{# minuut} other{# minuten}}',
+    '{0, plural, =1{# second} other{# seconds}} ago' => '{0, plural, =1{# seconde} other{# secondes}}',
+    'DISBOARD was made to make everyone enjoy Discord more.<br>We hope you find awesome Discord servers and friends here 😊.' => 'DISBOARD was gemaakt om Discord plezieriger te maken voor iedereen.<br>We hopen dat je hier geweldige Discord servers en vrienden vindt 😊.',
+    /* Welcome Messages on home page */
+    'Discover <span class="discord-logo">Discord</span> Servers' => 'Ontdek <span class="discord-logo">Discord</span> Servers',
+    'We connect <strong>Discord servers</strong> and <b>people</b> together' => 'Wij verbinden <strong>Discord servers</strong> en <b>mensen</b> samen',
+    'Public <span class="discord-logo">Discord</span> Servers' => 'Publieke <span class="discord-logo">Discord</span> Servers',
+    '<strong class="brand">DISBOARD</strong> is the place where you can <strong>list/find Discord servers</strong>.' => '<strong class="brand">DISBOARD</strong> is dè plek waar jij <strong>Discord servers kan plaatsen en zoeken</strong>.',
+    'Find and join some awesome servers listed here.' => 'Zoek en verbind wat gewelige servers vermeld hier.',
+    'Or <a href="{login-url}">login</a> and <strong>add your server if you are a server owner</strong> 😎' => 'Of <a href="{login-url}">meld aan</a> en <strong>voeg je eigen server toe als je de eigenaar bent</strong> 😎',
+    'Add Your Server!' => 'Voeg Je Server Toe!', // The large button
+    /* Categories */
+    'Adventure Games' => 'Avontuur Spellen',
+    'All Games' => 'Alle Spellen',
+    'Anime · Manga' => 'Anime · Manga',
+    'Community' => 'Sociaal',
+    'Computers' => 'Computers',
+    'FPS Games' => 'FPS Spellen',
+    'MMO Games' => 'MMO Spellen',
+    'MOBA Games' => 'MOBA Spellen',
+    'Movies' => 'Films',
+    'Music' => 'Muziek',
+    'Other' => 'Andere',
+    'Party Games' => 'Gezelschap Spellen',
+    'Programming' => 'Programmering',
+    'Puzzle Games' => 'Puzzel Spellen',
+    'RPG Games' => 'RPG Spellen',
+    'Retro Games' => 'Retro Spellen',
+    'Sandbox Games' => 'Sandbox Spellen',
+    'Simulation Games' => 'Simulatie Spellen',
+    'Sports Games' => 'Sport Spellen',
+    'Strategy Games' => 'Strategie Spellen',
+    'Technology' => 'Technologie',
+    'TCG Games' => 'TCG Games',
+    'Tabletop Games' => 'Bord Spellen',
+    'Fighting Games' => 'Vecht Spellen',
+    /* Meta Descriptions */
+    'DISBOARD is the public Discord server listing community. You can search Discord servers by your interest like Gaming, Anime, Music, etc. Find and join some awesome servers listed here!' => 'DISBOARD is dè publieke Discord server lijst gemeenschap. Je kan Discord servers zoeken op interesse zoals Gamen, Anime, Muziek, etc. Vind en sluit aan tot geweldige server vermeld hier!',
```

Comment:
The last sentence is impossible to salvage because Dutch syntax fucks over the original English setup.

'Vind hier een geweldige server om je bij aan te sluiten!' covers the intent of the original message and flows properly.